h/indexer/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from h.indexer.reindexer import reindex
```

Comment:
The [CLI code](https://github.com/hypothesis/h/blob/334c5b33b8ea80c6ef01fffb3f793c0faae28459/h/cli/commands/search.py#L31) is.